src/goto-analyzer/build_analyzer.h

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************\
+
+Module: Goto-Analyser Command Line Option Processing
+
+Author: Martin Brain, martin.brain@cs.ox.ac.uk
+
+\*******************************************************************/
+
+/// \file
+/// Build an abstract interpreter configured by the options.
+/// This may require options for:
+///  * which interpreter to use
+///  * which history to use
+///  * which storage to use
+///  * which domain to use
+///  * how that domain is configured
+/// Not every combination of options is supported.
+/// Unsupported options will give null.
```

Comment:
⛏️ Shouldn't this be documentation to the function below?

Reply:
Moved.